CONTRIBUTING.rst

```diff
@@ -247,43 +204,19 @@ Title template: *About [subject]* (if this is a background information for a tas
-
-   Explain what the user will be building in this tutorial. Include the problem we are solving and the components or tools that will be used to solve it. Remember that the example "problem" can be silly as long as it is easy to understand. Random pet name generator, office tea rota, plant watering system ... you decide. Whatever it is, the user knows where they are going and what they will see when they get there.
-
-   Pre-requisites
-   --------------
-
-   If there are tools, accounts, licenses or particular dependencies that need to be installed before we start, let's get those out of the way first. If it requires Python 22 and the user doesn't live in the future, they know not to start spinning up all the other pieces of the puzzle. Don't forget to prompt the user to sign up for Aiven with the free trial, if appropriate.
-
-   Bear in mind the expected experience level of the user. Our experienced technical audience does not need help installing python, handling dependencies, understanding what a text editor is. You should assume that level of knowledge and move on; the user will either enjoy getting into the interesting bits more quickly, or spend time filling in the gaps in their knowledge with materials from other excellent websites.
-
-   First component
-   ---------------
-
-   Introduce the first piece of the puzzle. Tutorials need to be as logical as possible, which means making as few steps, or as few context changes as possible. For example, don't download a data file, create a database, install some code dependencies, import the data into the file, set some code up locally and then go back to the database for credentials. Instead, make the database, get and import the data, then set up the code with dependencies and connection information to build the app.
-
-   At each stage, if the user can DO something, that builds confidence and a sense of achievement. So if there's a "SHOW TABLES" command or something else to affirm that something is going well, have them do that, and explain how it fits the big picture, before you move on.
+Limited availability note template
+'''''''''''''''''''''''''''''''''''
 
-   Second component
-   ----------------
+For features that are in the limited availability stage, add the following admonition directly undert the article title:
 
-   There will be many sections to a tutorial. Remember that titles should be things like "Create some initial data" or "Set up the indexes". These subheadings are shown in the secondary navigation and really help a user to navigate through a longer tutorial, either looking for something particular or just trying to keep track of where they got up to. If you are adding code snippets, make sure that it is clear with each snippet where in the project it should be added, and what its function is.
+.. important:: 
+    {feature name} is a limited availability feature. If you're interested in trying this feature out, contact your account team or Aiven support.
```

Comment:
For consideration:
1. `trying this feature out` vs`trying out this feature`
2. Aiven support vs \`the Aiven Support team <mailto:support@aiven.io>\`_